src/test/resources/features/cucumber.feature

```diff
@@ -0,0 +1,26 @@
+Feature: Test CRUD methods in Sample Employee REST API testing
+  Scenario: Save employee
+    Given  Valid employee object
+    When I  send a  request a new employee is created in db
+    Then I should get that employee object returned
+
+  Scenario:I want to get a employee based on id
+    Given id=75
```

Comment:
When you create a feature file, it should be generic in language. It should not have anything specific as id = 75

Check out the comments on this PR about the feature files https://github.com/vabav007/welcomeRepo/pull/5

Reply:
addresses in https://github.com/Jashika/springBootRestExample/pull/2/commits/9e9a7aac750f7c921ed6227482cdfeaee316c12a